StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentMethodFormViewController.swift

```diff
@@ -27,7 +29,9 @@ class PaymentMethodFormViewController: UIViewController {
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        _ = self.form.beginEditing()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```

Comment:
👍 nit: maybe add a comment about why we do this

Reply:
I'll follow up in another PR - I might remove/rework this.